test/Ocelot.AcceptanceTests/AggregateTests.cs

```diff
@@ -598,6 +598,81 @@ public void Should_return_response_200_with_user_forwarding()
             }
         }
 
+        [Fact]
+        [Trait("Bug", "2039")]
+        public void Should_return_response_200_with_body_sent_on_multiple_services()
```

Comment:
Excellent! However, unit tests are also necessary.

The testing workflow should be as follows:
- Conduct unit testing.
- Proceed to acceptance testing if unit testing is successful.
- Consider benchmark testing if acceptance testing is successful. While this is optional, it is highly recommended for system redesign.

Reply:
Duly noted. Let me rework that and get back to you with a more satisfying PR.